tsconfig.json

```diff
@@ -4,15 +4,14 @@
     "target": "ES2020",
     "module": "ES2020",
     "moduleResolution": "node",
+    "strict": true,
+    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "noImplicitAny": false,
+    "resolveJsonModule": true,
+    "noUnusedLocals": true,
     "removeComments": true,
+    "noImplicitAny": false,
     "preserveConstEnums": true,
-    "sourceMap": false,
-    "resolveJsonModule": true,
-    "types": ["@types/node"],
-    "strict": true,
-    "noUnusedLocals": true
   },
-  "include": ["src/**/*", "test/**/*"]
+  "include": ["src/**/*"]
```

Comment:
if I remember well the test part is needed otherwise the TS tests do not get build.

Reply:
We run our tests with `ts-jest` so they don't need to be compiled with the src, you will see in the CI for this PR our tests run fine